templates/python/setup.cfg

```diff
@@ -1,30 +0,0 @@
-[flake8]
-exclude =
-    .git,
-    __pycache__
-max-complexity = 10
-# black docs regarding flake8: https://black.readthedocs.io/en/stable/guides/using_black_with_other_tools.html#flake8
-# black enforces an equal amount of whitespace around slice operators. It is not PEP8 compliant.
-# black and flake8 also disagree on line length
-extend-ignore = 
-    # E203: Whitespace before ':'
-    E203,
-    # E501: Line too long
-    E501,
-
-[isort]
-profile = black
-
-[mypy]
-namespace_packages = True
-show_error_codes = True
-disable_error_code = import
-# do not require explicit `return None` in functions with Optional return type
-warn_no_return = False
-# don't skip functions which are unannotated, such as tests
-check_untyped_defs = True
-
-[tool:pytest]
-testpaths =
-    tests
-addopts = --cov=package-name --cov-report term-missing
```

Comment:
Is it intentional to remove this without replacement? 

You can also add such options to pyproject.toml (I guess then they will only be respected when running `poetry run pytest` and not with bare `pytest` though?). I have this in another project:
```
[tool.pytest.ini_options]
markers = ["live: test to run a live server running on localhost:8080"]
addopts = "-m 'not live'"
```

Reply:
good point, I overlooked it. Will add those to pyproject.toml